### ISSUE TITLE
[css-grid] calc function not supported by grid-gap

### ISSUE THREAD

**Issue by spAnser** [postcss/autoprefixer#1190]
An example of
```css
/* autoprefixer grid: on */
.grid {
  grid-template: 'col1 col2 col3 col4' 1fr / 1fr 1fr 1fr 1fr;
  gap: calc(20px - 4px);
}
```

The above code turns into:
```css
.grid {
  -ms-grid-rows: 1fr;
  -ms-grid-columns: 1fr calc 1fr calc 1fr calc 1fr;
      grid-template: 'col1 col2 col3 col4' 1fr / 1fr 1fr 1fr 1fr;
  grid-gap: calc(20px - 4px);
  gap: calc(20px - 4px);
}
```

However the correct output should be:
```css
.grid {
  -ms-grid-rows: 1fr;
  -ms-grid-columns: 1fr calc(20px - 4px) 1fr calc(20px - 4px) 1fr calc(20px - 4px) 1fr;
      grid-template: 'col1 col2 col3 col4' 1fr / 1fr 1fr 1fr 1fr;
  grid-gap: calc(20px - 4px);
  gap: calc(20px - 4px);
}
```

I am aware that the `calc` function used in this example is pointless.

**Comment by Dan503** [comment]
Can the issue name on this be changed to:
"[css-grid] calc function not supported in grid-gap".

**Comment by ai** [comment]
It is faster to fox it :) 76127d1

**Comment by ai** [comment]
Released in 9.4.8
